batch/test/test_batch.py

```diff
@@ -1156,7 +1156,12 @@ def test_cant_submit_to_default_with_other_ns_creds(client: BatchClient, remote_
             f'''
 python3 -c \'{script}\'''',
         ],
-        env={'HAIL_DOMAIN': DOMAIN, 'HAIL_DEFAULT_NAMESPACE': 'default', 'HAIL_LOCATION': 'external'},
+        env={
+            'HAIL_DOMAIN': DOMAIN,
```

Comment:
Can you clarify? This is intentionally set to the production domain.

Reply:
Sorry, the name "HAIL_DOMAIN" was elsewhere changed to "HAIL_PRODUCTION_DOMAIN", is this one intentionally unchanged?

Reply:
yes. this line in the job env must be `HAIL_DOMAIN` because that's what `deploy_config.py` picks up. The change to `HAIL_PRODUCTION_DOMAIN` was just to clarify the environment variable supplied to the pytest test, that it should be the domain associated with the production instance.